src/CommandingOfficers/Qis.java

```diff
@@ -97,7 +97,7 @@ public void modifyMoveType(UnitContext uc)
       // Watery terrain costs +1 move
       for( TerrainType terrain : TerrainType.TerrainTypeList )
       {
-        if( terrain.isWater() )
+        if( terrain.isWater() || terrain == TerrainType.RIVER )
```

Comment:
Hmm. We probably have a good reason for isWater() == false for rivers, since it seems counterintuitive. Do you remember why?

Reply:
Sea units can't go there.
The flag probably should be called... `OCEAN`?
So `SEA` tiles are a kind of `OCEAN`, as well as reefs and shoals.